pkg/auth/auth.go

```diff
@@ -134,7 +135,7 @@ func SignedInUser(ctx *macaron.Context, sess session.Store) (*models.User, bool)
 					}
 					return nil, false
 				}
-
+				ctx.Data["IsApiToken"] = true
```

Comment:
Here user uses basic auth, why should "IsApiToken" be true as well?

Reply:
This is not required now. At the very beginning I thought about having another middleware for all api routes that need authentication, but later I found it is sufficient to check it in reqToken.

Reply:
I see, then we could remove this line.